cmd/block-generator/generator/generate.go

```diff
@@ -18,19 +19,24 @@ import (
 
 var errOutOfRange = fmt.Errorf("selection is out of weighted range")
 
+type txID string
```

Comment:
I think `txType` would be more appropriate. "Transation ID" often refers to the transaction hash.